control-plane/subcommand/server-acl-init/command.go

```diff
@@ -804,6 +764,90 @@ func (c *Command) createAuthMethod(consulClient *api.Client, authMethod *api.ACL
 		})
 }
 
+type gatewayRulesGenerator func(name, namespace string) (string, error)
+
+// ConfigureGatewayParams are parameters used to configure Ingress and Terminating Gateways.
+type ConfigureGatewayParams struct {
+	//GatewayType specifies whether it is an ingress or terminating gateway
```

Comment:
nit
```suggestion
	// GatewayType specifies whether it is an ingress or terminating gateway.
```

I'm surprised it didn't get `go fmt`'d, but maybe it doesn't look at struct fields?